client.py

```diff
@@ -51,4 +51,22 @@ async def on_ready():
 async def on_member_join(member):
     role = get(member.guild.roles, name='UNVALIDATED')
     await member.add_roles(role)
-    
\ No newline at end of file
+
+
+@bot.event
+async def on_message(message):
+    emoji_box = list("🟥🟧🟨🟩🟪🔴🟠🟡🟢🟣")
+
+    if message.content.startswith('/poll'):
```

Comment:
[Discord.py provides a command module](https://discordpy.readthedocs.io/en/latest/ext/commands/commands.html) , so you don't have to manually parse the command and instead just take the arguments and start parsing from there